docs/Guides/Porting.md

```diff
@@ -173,7 +164,7 @@ The `-v` option above specifies verbose output.
 
 Once finished, you will notice that your project was built and installed under `$HOME/zopen/prod/jq`.
 
-*Creating Patches*
+#### Creating Patches
```

Comment:
The example below would be worth discussing. My preference is that we have individual file patches so that:
a) if someone else needs to make another mod to one of the files already patched, they don't have to remove the file from the first patch and/or assume an ordering to how the patches are applied
b) it is easier to see the full extent of changes
c) it is easier to create a README.md (which we should likely start doing to keep track of what these patches are) for each set of changes in a file (alternately for this, a set of file patches could be put into a directory with a README.md for the set of changes in a directory if that's more reasonable)